### PR TITLE
UT_Alignment_12d_5

### PR DESCRIPTION
**Pull request by LeeGregory12d**
# IFC4.3 RC2 Unit test
IfcAlignment with HG: left and right arc, and leading and trailing clothoid transitions on each arc.
No VG.

## Review requested
- leading and trailing clothoid transitions on left and right arcs
@pjanck

## Instructions
 Added new UT <UT_Alignment_12d_5> with supporting documentation

### CODE REVIEW

**Comment by larswik** [comment]
File looks fine to me! I am not sure why the property set containing the global id for the alignment is needed?

**Comment by LeeGregory12d** [comment]
Sorry - the property set is an internal thing and was there because I forgot to tick the flag in **12d Model** to not output it.

File: IFC 4.3/UT_Alignment_12d_5/UT_Alignment_12d_5.ifc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#42 = IFCCARTESIANPOINT((172.607799056118, 58.8789936654367));
+#43 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #44, 9.72132306240745E-1, -0., -75., 10., $, .CLOTHOID.);
+#44 = IFCCARTESIANPOINT((199.242296489501, 97.922357538628));
+#45 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #46, 9.05465639574999E-1, -75., -75., 25.7981600209167, $, .CIRCULARARC.);
+#46 = IFCCARTESIANPOINT((205.058702884133, 106.054387861243));
+#47 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #48, 5.61490172629377E-1, -75., -0., 30., $, .CLOTHOID.);
+#48 = IFCCARTESIANPOINT((224.128542220608, 123.240134547601));
+#49 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #50, 3.61490172629377E-1, 0., 0., 54.4255329033339, $, .LINE.);
+#50 = IFCCARTESIANPOINT((251.3723118619, 135.673231025453));
+#51 = IFCPROPERTYSET('2vWb8GmYP8F8FdxwLGGT3G', #2, '12d Model', $, (#52));
```
Comment (larswik) [inline IFC 4.3/UT_Alignment_12d_5/UT_Alignment_12d_5.ifc:58]: Maybe remove these last three lines to keep it as a "unit test" :-)

**Review by larswik — APPROVED** [review]
I don't think that the pSet hurts, so approving...

**Comment by LeeGregory12d** [comment]
UT moved to repository IFC-infra-unit-test

**Pull request closed without merging**